File: UFAM/PLP/codes/questao04.pl
open(my $file, "<", "alunos_IFPB.csv") or die "Não foi possível abrir o arquivo: $!";

while (my $line = <$file>) {
    chomp $line;

    my @campos = split(",", $line);
    my $matricula = $campos[1];
    my $cidade = $campos[4];

    if ($cidade =~ /JOÃO PESSOA/) {
        print "Matricula: $matricula\n";
    }
}

close($file);

File: UFAM/PLP/codes/questao04.py
file = open("alunos_IFPB.csv","r") 

line = file.readline() 

while line:

    
    campos = line.split(",")
    #print(campos)
    matricula=campos[1]
    cidade = campos[4]
 
    if"JOÃO PESSOA" in cidade:
        print("Matricula:", matricula) 

    line  = file.readline()

file.close( )